Expo1900/Expo1900/Controller/AppDelegate.swift

```diff
@@ -8,8 +8,15 @@ import UIKit
 
 @main
 class AppDelegate: UIResponder, UIApplicationDelegate {
-
-
+    var changeOrientation: Bool = true
+    
+    func application(_ application: UIApplication, supportedInterfaceOrientationsFor window: UIWindow?) -> UIInterfaceOrientationMask {
+        if !changeOrientation {
+            return [.all]
+        } else {
+            return [.portrait]
+        }
```

Comment:
조건문의 컨디션은 긍정일 때 읽기 좋습니다.

또한 `changeOrientation`는 여러가지 해석의 여지가 있으니(기준이 portrait인지 landscape인지 모르니까) 다른 명확한 이름을 고민해주세요. 

```suggestion
        if changeOrientation {
            return [.portrait]
        } else {
            return [.all]
        }
```

Reply:
- https://github.com/yagom-academy/ios-exposition-universelle/pull/286/commits/29c18c71ab61bfdac6287b1b1484de05a21fe745

`isOnlyPortrait`으로 네이밍을 수정하였습니다.😊